client/src/components/App.jsx

```diff
@@ -15,58 +15,85 @@ import Navbar from './Navbar.jsx';
 import Friend from './Friend.jsx';
 import StockDetailPage from './StockDetailPage.jsx';
 import Searchbar from './Searchbar.jsx';
+import { sampleState } from '../../../sampleData/sampleState.js';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {
-      user: {
-        first_name: '',
-        last_name: '',
-        username: '',
-        email: '',
-        cashBalance: 0,
-        rank: null,
-        userPortfolio: [
-          // {
-          //   stockName:
-          //   sharesOwned
-          // }
-        ],
-        friends: [
-          // username, username
-        ]
-      },
-      sessionInfo: {}, // need to be populated with passport
-      selectedFriend: {
-        username: '',
-        rank: 0,
-        portfolioValue: 0,
-        selectedFriendPortfolio: [
-        //  {
-        //     stockName:
-        //     sharesOwned
-        //   },
-        //   {}
-        ]
-      },
-      stockSelected: {
-        name: 'Tesla',
-        symbol: 'TSLA',
-        price: 100,
-        data: [
-          // {},{}
-        ]
-      }
-    };
+    this.fetchSelectedStock = this.fetchSelectedStock.bind(this);
+    this.state = sampleState;
+    // {
+    //   user: {
+    //     first_name: '',
+    //     last_name: '',
+    //     username: 'bezos_the_first',
+    //     email: '',
+    //     cashBalance: 200000,
+    //     rank: 1,
+    //     userPortfolio: [
+    //       {
+    //         stockName: 'Amazon.com, Inc.',
+    //         stockSymbol: 'AMZN',
+    //         valueOwned: 350000
+    //       },
+    //       {
+    //         stockName: 'Telsa, Inc.',
+    //         stockSymbol: 'TSLA',
+    //         valueOwned: 300000
+    //       },
+    //       {
+    //         stockName: 'Apple',
+    //         stockSymbol: 'AAPL',
+    //         valueOwned: 200000
+    //       },
+    //       {
+    //         stockName: 'StockDucks, Inc.',
+    //         stockSymbol: 'STKD',
+    //         valueOwned: 200000
+    //       }
+    //     ],
+    //     friends: [
+    //       // username, username
+    //     ]
+    //   },
+    //   sessionInfo: {}, // need to be populated with passport
+    //   selectedFriend: {
+    //     username: '',
+    //     rank: 0,
+    //     portfolioValue: 0,
+    //     selectedFriendPortfolio: [
+    //     //  {
+    //     //     stockName:
+    //     //     sharesOwned
+    //     //   },
+    //     //   {}
+    //     ]
+    //   },
+    //   stockSelected: {
+    //     name: 'Tesla',
+    //     symbol: 'TSLA',
+    //     price: 100,
+    //     data: [
+    //       // {},{}
+    //     ]
+    //   }
+    // };
 
-    this.fetchSelectedStock = this.fetchSelectedStock.bind(this)
+    this.fetchSelectedStock = this.fetchSelectedStock.bind(this);
     this.handleTrade = this.handleTrade.bind(this);
+    // this.initialize();
   }
 
   componentDidMount() {
     this.getCurrentUser();
   }
+  initialize() {
```

Comment:
Same here. Can you remove initialize? Totally awesome how you you were mocking the data though!

Reply:
Agree here too, but also not mine.